rllib/algorithms/algorithm.py

```diff
@@ -551,6 +555,13 @@ def setup(self, config: PartialAlgorithmConfigDict):
                 logdir=self.logdir,
             )
 
+            if self.config["evaluation_with_async_requests"]:
```

Comment:
Yeah, but all our eval settings start with `evaluation_...`
But fair enough, now that we have config objects with type-safe property names and method args, this may not be so much of a problem anymore :)

Reply:
done

Reply:
We should make this the default as fast as possible to not get users confused as of the difference between `evaluation_parallel_to_training` and `enable_async_evaluation`.